extensions/schema-compare/src/telemetry.ts

```diff
@@ -0,0 +1,113 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Suggest putting this into a separate telemetry subfolder to keep all the telemetry related stuff in one place

Reply:
Eh, on second thought feel free to ignore this. It's small enough I'm not too concerned